src/bsp/src/stm32/hal/stm32f429zi/USB_DEVICE/App/usbd_desc.c

```diff
@@ -65,9 +65,9 @@
 
 #define USBD_VID 1155
 #define USBD_LANGID_STRING 1033
-#define USBD_MANUFACTURER_STRING "STMicroelectronics"
-#define USBD_PID_FS 22336
-#define USBD_PRODUCT_STRING_FS "STM32 Virtual ComPort"
+#define USBD_MANUFACTURER_STRING "SwarmUS"

```

Comment:
Was this modified by you or inside CubeMX? Because this will be overwritten if it is not part of the .ioc

Reply:
Changed it to be modified by the Cube file